lib/structured_data/utilities.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module StructuredData
+  class Utilities
+    def self.find_dependents(file_number_or_ssn)
+      service = LighthouseBGS::Services.new(external_uid: 0, external_key: 'VA.gov')
```

Comment:
Do we need specific "external" parameters here?

Reply:
These keys should be specific to the person/veteran who is making the call. The external_uid should be the users icn number and the external_key should be the users email address.

Reply:
@jjulian @charleystran this form can be filled out by anonymous users so we may not have that info. what should we fill out in that case?   The BGS calls will fail if nil values are provided for these uids

Reply:
One thing we could do is pass the claimant email and phone number if we really want to keep track of that these calls relate to. I have a message out to the BGS team to ask them how this information is being used on their end

Reply:
Its a BGS requirement that we pass enough information in these values to identify the end user making the call. Its all a part of their logging and audit trail. They want to be able to track down any user who made the call. Something outside our control. Maybe email and phone number would be enough? We would need to check with BGS on that.

Reply:
@charleystran @johnpaulashenfelter I followed up with Cory Easley from BGS on this. I can confirm that it is used for logging/audit trail, no other use. For this burials form, an anonymous user will be able to fill out the form. I suggest we use the Claimant email and phone number for this purpose as they are the ones who are submitting the form to VA.gov

Reply:
Did Corey say he was ok with using those values? I know when we setup to use icn and email address, there were a couple people like John Dell that had to give it the all clear. If they are ok with sending those values in those headers, then I am ok with it.

Reply:
Will ask John Dell in BGS email thread to confirm and update the PR accordingly

Reply:
Is the UID numeric-only? I'm considering collisions between a 10-digit claimant phone number and an ICN number.
If UID is a string, we can make phone numbers unique with a prepended string, like `+` or `tel:`

Reply:
Following up @bshyong did you get a reply from the BGS email thread/John Dell?